### ISSUE TITLE
Verify, enrich and export metadata

### ISSUE THREAD

**Issue by nesnoj** [openego/eGon-data#451]
There're some final steps to be done on all the metadata created in the pipeline #XXX:

- [ ] Verify metadata: check metadata (written as table comments) of a predefined list of tables (all which are ticked at `Metadata needed?` in #XXX).
If something does not comply, should we mark the task failed or throw a warning? (I'd recommend the latter)
- [ ] Update / set missing fields
  - `id` (URL path to resource on OEP)
  - `publicationDate` (`now()`)
- [ ] Add internal version no of dataset? (from `metadata.datasets`)
- [ ] Export as JSON to some central place. What about bibtex, would that be helpful too?

**Comment by jh-RLI** [comment]
The oemetadata validation implementation is now worked on in this issue: https://github.com/openego/eGon-data/issues/1305

**Comment by jh-RLI** [comment]
@nesnoj can you clarify again what the difference between the metadata here https://github.com/openego/eGon-data/tree/dev/src/egon/data/metadata

and the metadata which is talked about in issue https://github.com/openego/eGon-data/issues/450 is? 

As far as i understand, for now i focus on what is in the metadata directory. Can i view the eGon list as "now outdated"?

**Comment by nesnoj** [comment]
> [@nesnoj](https://github.com/nesnoj) can you clarify again what the difference between the metadata here https://github.com/openego/eGon-data/tree/dev/src/egon/data/metadata
> 
> and the metadata which is talked about in issue [#450](https://github.com/openego/eGon-data/issues/450) is?
> 
> As far as i understand, for now i focus on what is in the metadata directory. Can i view the eGon list as "now outdated"?

Yes! The files in https://github.com/openego/eGon-data/tree/dev/src/egon/data/metadata are the results from #450 so the issue can be closed.
You can understand the files in the metadata directory as the minimum for now which will be extended later.
